src/CreeDictionary/phrase_translate/management/commands/translatewordforms.py

```diff
@@ -87,187 +41,67 @@ def __str__(self):
-        else:
-            wordform_filter = {}  # No restriction
-
-        logger.info("Removing existing auto-definitions")
-        deleted_rows, deleted_row_details = Definition.objects.filter(
-            citations__abbrv="auto", **wordform_filter
-        ).delete()
-        logger.info(
-            f"Deleted {deleted_rows:,} rows for auto-definitions: {deleted_row_details}"
-        )
-
-        (ds, created_) = DictionarySource.objects.get_or_create(abbrv="auto")
-
-        definition_buffer = InsertBuffer(manager=Definition.objects, assign_id=True)
-        citation_buffer = InsertBuffer(manager=Definition.citations.through.objects)
-
-        for _, definitions in self.generate_translations():
-            for auto_definition in definitions:
-                definition_buffer.add(auto_definition)
-                citation_buffer.add(
-                    Definition.citations.through(
-                        dictionarysource_id=ds.abbrv,
-                        definition_id=auto_definition.id,
-                    )
-                )
-
-        definition_buffer.save()
-        citation_buffer.save()
-        logger.info("Translation done")
+class Command(BaseCommand):
+    help = """XXX No longer works
+
+            This command no longer works. It required that all wordforms be
+            instantiated in the database, even if they weren’t translatable, and
+            running it multiple times could result in multiple copies of
+            auto-definitions.
+            
+            Use importjsondict instead with the --translate-wordforms option
+            instead.
+        """
 
-    def write_translations_to_jsonl(self, filename):
-        with open(filename, "w") as out:
-            for wordform, auto_definitions in self.generate_translations():
-                out.write(
-                    json.dumps(
-                        {
-                            "wordform_text": wordform.text,
-                            "wordform_analysis": wordform.analysis,
-                            "wordform_lemma_text": wordform.lemma.text,
-                            "wordform_lemma_analysis": wordform.lemma.analysis,
-                            "translations": [
-                                (
-                                    d.auto_translation_source.text,
-                                    ",".join(
-                                        c.abbrv
-                                        for c in d.auto_translation_source.citations.all()
-                                    ),
-                                    d.text,
-                                )
-                                for d in auto_definitions
-                            ],
-                        },
-                        ensure_ascii=False,
-                    )
-                    + "\n"
-                )
+    def add_arguments(self, parser: ArgumentParser):
+        pass
```

Comment:
You can likely remove this function entirely. I don't think it's necessary for the `BaseCommand` class.